### ISSUE TITLE
Layout/ClosingParenthesisIndentation and Layout/CommentIndentation allegedly do not support IndentationWidth parameter

### ISSUE THREAD

**Issue by Timmitry** [rubocop/rubocop#15137]
### Before submitting

- [x] I have searched the existing issues and this hasn't been reported yet.
- [x] I am running the latest stable version of RuboCop.

### Issue type

Other

### Cop name

Layout/ClosingParenthesisIndentation,Layout/CommentIndentation

### Steps to reproduce the problem

Test file (uses tabs, not spaces!)

```rb
class Test
	expect(something).to eql(
		foo: 123
	)
	# This is a comment
end
```

### Expected behavior

Rubocop either does not show any errors here (with Layout/ClosingParenthesisIndentation and Layout/CommentIndentation commented out in the config), or I can configure these two cops with indentation width.

### Actual behavior

Rubocop gives these errors:

Layout/ClosingParenthesisIndentation: Indent `)` to column 0 (not 1)
Layout/CommentIndentation: Incorrect indentation detected (column 1 instead of 2).

When I comment in the whole configuration, it does not give errors in the file, but when running the rubocop CLI:

```
Warning: Layout/ClosingParenthesisIndentation does not support IndentationWidth parameter.

Supported parameters are:

  - Enabled

Warning: Layout/CommentIndentation does not support IndentationWidth parameter.

Supported parameters are:

  - Enabled
  - AllowForAlignment
```

However, the IndentationWidth seems to do exactly what I wanted and fixes the problems!

### RuboCop configuration

```yaml
# Layout/ClosingParenthesisIndentation:
#   IndentationWidth: 1

# Layout/CommentIndentation:
#   IndentationWidth: 1

Layout/IndentationStyle:
  EnforcedStyle: tabs

Layout/FirstArgumentIndentation:
  IndentationWidth: 1
```

### RuboCop version

```shell
1.86.1 (using Parser 3.3.11.1, Prism 1.9.0, rubocop-ast 1.49.1, analyzing as Ruby 4.0, running on ruby 4.0.1) [arm64-darwin24]
```

**Comment by Timmitry** [comment]
Thanks, that was fast 💚
